src/phases/execution/clusteredCollection.yml

```diff
@@ -0,0 +1,128 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/server-execution"
+Description: >
+  Run basic insert and find and delete workload on a collection clustered by {_id: 1} .
+  Clustered collections are planned to mainly serve operations over the cluster
+  key such as monotonically increasing inserts, range queries and range deletions.
+
```

Comment:
It looks like you may also have missed my earlier comment about adding keywords:

```
Keywords:
- indexes
- clustered
```